src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportWorkloadsCrossTarget.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.ImportWorkloadsCrossTarget.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+============================================================
+                                     _FindAllReferenceWorkload
+
+ Collecting all Workload requirement for each inner build.
+============================================================
+-->
+
+  <Target Name="_FindAllReferenceWorkload"
+          DependsOnTargets="_ComputeTargetFrameworkItems"
+          Returns="@(_ResolvedSuggestedWorkloads)">
+
+    <MSBuild Projects="@(_InnerBuildProjects)"
+             Condition="'@(_InnerBuildProjects)' != '' "
+             Targets="_FindAllReferenceWorkload"
+             BuildInParallel="$(BuildInParallel)">
+      <Output ItemName="_ResolvedSuggestedWorkloadsWithDuplicate" TaskParameter="TargetOutputs"/>
```

Comment:
@rainersigwald _ResolvedSuggestedWorkloadsWithDuplicate should collect all output of batches right?

Reply:
Correct.